src/time/src/mcp_server_time/server.py

```diff
@@ -0,0 +1,145 @@
+from datetime import datetime
+import json
+from typing import Dict, Any, Optional, Sequence
+
+import pytz
+from tzlocal import get_localzone
+from mcp.server import Server
+from mcp.server.stdio import stdio_server
+from mcp.types import Tool, TextContent, ImageContent, EmbeddedResource
+
+
+class TimeServer:
```

Comment:
It looks like this is duplicated across `server.py` and `server-resources.py`—was that intentional?

Reply:
Explained [here](https://github.com/modelcontextprotocol/servers/pull/44#discussion_r1858115345)